server/src/main/java/org/jetbrains/bsp/bazel/server/sync/languages/cpp/CppModule.kt

```diff
@@ -0,0 +1,18 @@
+package org.jetbrains.bsp.bazel.server.sync.languages.cpp
+
+import org.jetbrains.bsp.bazel.server.sync.languages.LanguageData
+
+data class CppModule(
+    val copts: List<String>?,
+    val defines: List<String>?,
+    val linkOpts: List<String>?,
+    val linkShared: Boolean
+    ) : LanguageData {
+
+
+    companion object {
+        @JvmStatic
+        fun fromLanguageData(languageData: LanguageData?) = languageData as? CppModule
```

Comment:
probably it wont be useful - it iis used in scala module in the bloop exporter - very specific usecase